tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -105,6 +106,7 @@ def test_payload(self) -> None:
         checkin = MonitorCheckIn.objects.get(guid=self.guid)
         # the expected time should not include the margin of 5 minutes
         assert checkin.expected_time == expected_time - timedelta(minutes=5)
+        assert checkin.trace_id is not None
```

Comment:
Should we store the uuid and confirm that they match?

Reply:
sure thing